compiler/passes/codegen.cpp

```diff
@@ -651,9 +684,9 @@ static void codegen_header() {
   //
   forv_Vec(FnSymbol, fn, gFnSymbols) {
     legalizeName(fn);
-    functions.add(fn);
+    functions.push_back(fn);
   }
-  qsort(functions.v, functions.n, sizeof(functions.v[0]), compareSymbol);
+  std::sort(functions.begin(), functions.end(), compareSymbol2);
```

Comment:
It seems to "just work".  Should I package it in a functor class to be safe, or do you think it is fine to leave it as is?